QUARKUS-874.md

```diff
@@ -0,0 +1,37 @@
+# QUARKUS-874 Production support for Windows
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-874
+
+The goal is to ensure some level of support and identify existing limitations on Quarkus side for Windows.
+
+## Scope of the testing
+The selected Test Suites will be executed on the selected Windows Platforms to ensure functionality is fully accomplished.
+In order to accommodate this in our current test suites, we need to rename existing jobs in Jenkins in order to differentiate them for those running on RHEL and on Windows platforms. 
+
+### Selected Windows Platforms
+- Windows Server 2019
```

Comment:
Please add a note that we will make sure to have access to Windows 10 machine in PSI when needed in case of fix for customer issue bound to that particular environment.

Reply:
Added